transports/noise/Cargo.toml

```
@@ -33,7 +33,8 @@ async-io = "1.2.0"
 env_logger = "0.9.0"
 libp2p-tcp = { path = "../../transports/tcp" }
 quickcheck = "0.9.0"
-sodiumoxide = "0.2.5"
+libsodium-sys-stable = { version = "1.19.22", features = ["fetch-latest"] }
```

```suggestion
libsodium-sys-stable = { version = "1.19.22" }
```

Could we do without this feature? I.e. would it be fine to not use the latest libsodium?

---

It's the default - I am just explicitly mentioning here as to the intent - and it fetches the latest stable.

Otherwise it's optimized or minimal - I recommend the stable

Cargo features:

    fetch-latest: download the latest stable version.
    optimized: build a version optimized for the current platform.
    minimal: do not build deprecated APIs.